packages/aggregator/src/aggregationTypes.js

```diff
@@ -13,6 +13,7 @@ export const AGGREGATION_TYPES = {
   FINAL_EACH_MONTH: 'FINAL_EACH_MONTH',
   FINAL_EACH_QUARTER_FILL_EMPTY_QUARTERS: 'FINAL_EACH_QUARTER_FILL_EMPTY_QUARTERS',
   FINAL_EACH_QUARTER: 'FINAL_EACH_QUARTER',
+  FINAL_EACH_WEEK_FILL_EMPTY_WEEKS_WITH_NULL: 'FINAL_EACH_WEEK_FILL_EMPTY_WEEKS_WITH_NULL',
```

Comment:
As discussed, we might wanna just make this `FINAL_EACH_WEEK_FILL_EMPTY_WEEKS` and override the default of `previous` with `null` in the aggregation config